apps/prairielearn/elements/pl-matching/pl-matching.css

```diff
@@ -47,3 +47,38 @@
   background-color: rgba(0, 0, 0, 0.03);
   margin-right: 0.5rem;
 }
+
+.pl-matching-container-right {
+  display: flex;
+}
+
+.pl-matching-container-right .pl-matching-statements {
+  flex: 0.2 0 25%;
+  display: flex;
+  flex-direction: column;
```

Comment:
These are not necessary because this is already set for `pl-matching-statements` above.